src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -667,6 +670,13 @@ const formConfig = {
                         confirmationCode: response.data.attributes.guid
                       };
                     },
+                    attachmentSchema: {
+                      'ui:title': 'Document type',
+                      'ui:options': {
+                        groupBy: true,
```

Comment:
How does the `groupBy` ui:option work?

Reply:
Good catch, this is zombie code and should be removed.